cli/chain.go

```diff
@@ -872,6 +876,8 @@ var chainExportCmd = &cli.Command{
 			return fmt.Errorf("must specify filename to export chain to")
 		}
 
+		rsrs := cctx.Int64("recent-stateroots")
```

Comment:
We should probably require `recent-stateroots` to be greater than finality because some lookbacks are that long.